tests/MongoDB.Driver.Core.Tests/Specifications/server-discovery-and-monitoring/ServerDiscoveryAndMonitoringTestRunner.cs

```diff
@@ -482,7 +478,8 @@ protected override bool ShouldReadJsonDocument(string path)
             }
         }
     }
-	internal static class MultiServerClusterReflector
+
+    internal static class MultiServerClusterReflector
```

Comment:
whole changes in this file can be reverted since they are only about formatting, but I really wanted to fix this one. And since I've started, I've updated all other lines(automatically) a bit and renamed the whole class.